codec/codec-fury/src/main/resources/META-INF/services/sofa-rpc/com.alipay.sofa.rpc.codec.Serializer

```diff
@@ -0,0 +1 @@
+protobuf=com.alipay.sofa.rpc.codec.fury.FurySerializer
```

Comment:
u need refer to 
`com.alipay.sofa.rpc.codec.SerializerFactory#buildLoader` and 
`com.alipay.sofa.rpc.ext.ExtensionLoaderFactory#getExtensionLoader(java.lang.Class<T>)` 
to find how this file works. 

Such files in codec/hessian, codec/jackson and etc. may help u understanding.

Reply:
yes,i will to refer